850.split-ambiguities/y.yaml

```diff
@@ -6,6 +6,10 @@
 - { name: yali, wwwpart: pisi, setname: yali-installer }
 - { name: yali, addflag: unclassified }
 
+- { name: yass, wwwpart: kokkinizita, setname: yet-another-scrolling-scope }
```

Comment:
I try to follow these rules when picking unique names:
- keep the original name as a prefix for searchability
- keep the resulting name short for convenience, if possible
- try to convey the topic of the software for clarity, if possible

So IMO it'd be better to use `yass-scrolling-scope` and `yass-genomic-similarity` here.

Reply:
Good suggestion. I changed them.